### ISSUE TITLE
Post's archive feature is misleading (Disable Archive Feature)

### ISSUE THREAD

**Issue by AlbertoMontalesi** [forem/forem#4504]
Archived posts are still visibile on a user profile.
I've archived two of my posts but when I go to my dashboard I can still see them (I viewed it as logged out). 
That kind of defeats the purpose of archiving posts in the first place if they are still visible

**Comment by teaglebuilt** [comment]
I also see the same thing

**Comment by rhymes** [comment]
Would it be possible to have screenshots and/or the URLs of the articles you're referring to?

**Comment by AlbertoMontalesi** [comment]
Sure, it's very simple, this is my profile https://dev.to/albertomontalesi and if you scroll to my first two articles you can see this one https://dev.to/albertomontalesi/es6-discovering-sets-weaksets-maps-and-weakmaps-56lb which I've archived and reposted again here https://dev.to/albertomontalesi/es6-discovering-sets-weaksets-maps-and-weakmaps-db2


![Screenshot_2019-10-21-20-45-26-939_com android chrome](https://user-images.githubusercontent.com/23130353/67210745-d381dc00-f443-11e9-81bd-3d29094e64e7.png)

**Comment by maestromac** [comment]
Thanks for this issue @AlbertoMontalesi . 

This feature was originally implemented to hide unpublished post that came through RSS feed, and not meant to hide an already published post. It is misleading. You would have to unpublish the article in order to formally withdraw it from the public. I propose the solution here is to disable the "archive" feature on a published post.

**Comment by AlbertoMontalesi** [comment]
@maestromac ok that makes sense, I actually didn't even see the unpublish button until you mentioned it.

**Comment by mstruve** [comment]
As small of a change as this is I think the feature removal itself might warrant an RFC. @vaidehijoshi or @joshpuetz thoughts?

**Comment by vaidehijoshi** [comment]
@mstruve agreed.

> This feature was originally implemented to hide unpublished post that came through RSS feed
Not sure if hiding unpublished posts on the RSS feed is something that is actually being used by anyone, or if it is helpful. We may either want to make it very clear what the archive feature _does_ and hide it in places where it is misleading, OR, alternatively, we remove the feature entirely. I think it depends completely on whether anyone even uses it.

But either way, this is going to be a more widespread change and should be a proposal since it's clear now that it isn't a bugfix  👍🏽

**Comment by rhymes** [comment]
Some data on how many archived articles there are on DEV: https://dev.to/admin/blazer/queries/316-archived-articles-counts

**Comment by joshpuetz** [comment]
I'd argue in the short term the bug fix is to remove the `archive post` button from published articles: it shouldn't be there.

What would the RFC entail: removing the archive feature completely?

**Comment by vaidehijoshi** [comment]
> What would the RFC entail: removing the archive feature completely?

@joshpuetz Yeah, exactly. If we remove the archive _feature_ completely, it would warrant a RFC as it is removing a feature. 

But if we want to audit the feature and figure out a way to keep it in and find a light-touch code solution that clarifies what it is and how it works, that doesn't need to be a RFC and can be an issue.

**Comment by michael-tharrington** [comment]
This one recently came up again as an org admin was trying to archive one of their org's published posts. They didn't realize the feature was specifically meant to archive drafts and thought that they could use it to hide a published post from view.

I didn't actually realize what the archive feature was for and also assumed that it could be used to hide public posts. When I tried to hide a public post, it appeared to work, but upon refresh the post was still visible in my dashboard. See the following loom:

https://www.loom.com/share/9e0fb9f4222146a7a144255cbc7903cf

**Comment by michael-tharrington** [comment]
I actually got one more loom for y'all. This one is to show that archiving a post under an org is problematic because you can't easily unarchive it (in cases when the post isn't yours, it's unclear if you can unarchive it)

https://www.loom.com/share/d2df99e8d14844e0a450ce9957944ed4

**Comment by ch-stripe** [comment]
> This one recently came up again as an org admin was trying to archive one of their org's published posts. They didn't realize the feature was specifically meant to archive drafts and thought that they could use it to hide a published post from view.

Chiming in again that I found this thread because I tried exactly this and am actually still struggling to find the "unpublish" option.

**Comment by Xwixpacer** [comment]
https://trafficadbar.com/Maryannbustamante

**Comment by Xwixpacer** [comment]
https://trafficadbar.com/Maryannbustamante

**Comment by mikeydorje** [comment]
Reviewed `archived` column in `db/migrate/20190504015859_add_archived_to_articles.rb` and dashboard views.

The archive feature still works as designed—it hides unpublished articles from RSS feeds. It does not hide published posts from profiles.

Multiple comments (2019, 2022, 2023) confirm users find this misleading and expect it to hide published posts. The correct action for published posts is unpublishing, but users report the unpublish option is hard to find.

Valid UX issue—the interface doesn't clearly distinguish between "archive draft" (RSS control) and "unpublish post" (remove from public view).
